mtags/src/main/scala-3/scala/meta/internal/pc/HoverProvider.scala

```diff
@@ -157,24 +157,40 @@ object HoverProvider:
             findRefinement(info)
           case _ => ju.Optional.empty()
 
-      findRefinement(sel.tpe.termSymbol.info)
+      findRefinement(sel.tpe.termSymbol.info.dealias)
     case _ =>
       ju.Optional.empty()
 
 end HoverProvider
 
 object SelectDynamicExtractor:
-  def unapply(path: List[Tree]) =
+  def unapply(path: List[Tree])(using Context) =
     path match
+      case Select(_, _) :: Apply(
```

Comment:
Can you add short comment before each case what code will be matched be it? In other words, some clue for people who don't have ast explorer built-in in mind :D

Reply:
I've added annotations which tests match which case, because the examples aren't that small